docs/features/analysis.md

```diff
@@ -754,3 +756,48 @@ status:
   startedAt: "2021-02-10T00:15:26Z"
 ```
 
+### Empty array
+
+Metric providers can sometimes return empty array, e.g., no data returned from prometheus query.
+
+Here are two examples where a metric result of empty array is considered successful and failed respectively.
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: AnalysisRun
+  ...
+    successCondition: len(result) == 0 || result >= 0.95
```

Comment:
shouldn't it be `len(result) == 0 || result[0] >= 0.95` since `result` is an array